### ISSUE TITLE
Can't remove a block template

### ISSUE THREAD

**Issue by bleepsandblops** [verbb/vizy#145]
**Description**
I'm unable to remove a template for a block in a Vizy field.

**Steps to reproduce**
1. Create a new Vizy field, with a new block and assign it any template, save.
2. Reopen that field, delete the template path, save again
3. Reopen the field, the template path is still there

**Additional info**
- Plugin version: 1.0.13
- Craft version: 3.7.42
- Multi-site: No

**Additional context**
Not sure if this is related, but the reason I wanted to remove the template in the first place is that it looks like when I do a test on a Vizy field, for example:
`{% if article.contentVizy | length %}`
it's actually parsing the block templates somehow, as it's giving me errors like:
`Variable "node" does not exist.`
because I have a node variable in that block template...

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.14",
  "...": "..."
}
```

Then run `composer update`.

Let me know if that doesn't fix up that error!

**Comment by engram-design** [comment]
Fixed in [1.0.14](https://github.com/verbb/vizy/releases/tag/1.0.14)
